Apps/WebClient/src/ClientApp/src/components/report/ImmunizationHistoryReportComponent.vue

```diff
@@ -258,12 +258,18 @@ export default class ImmunizationHistoryReportComponent extends Vue {
 <template>
     <div>
         <section>
-            <b-row>
+            <b-row class="d-none d-md-block">
                 <b-col>
                     <h4>Immunization History</h4>
                 </b-col>
             </b-row>
-            <b-row v-if="isEmpty && !isLoading">
+            <b-row v-if="isEmpty && !isLoading" class="d-none d-md-block">
+                <b-col>No records found.</b-col>
+            </b-row>
+            <b-row
+                v-if="isRecommendationEmpty && isEmpty && !isLoading"
+                class="mt-2 d-sm-block d-md-none"
```

Comment:
`d-sm-block` won't cover the xs breakpoint; you would need `d-block` for that.  However, neither are needed here as the default for `<b-row>` (display: flex) is sufficient for the element to be rendered.

Reply:
Fixed